### ISSUE TITLE
[FR] file system link files

### ISSUE THREAD

**Issue by jmvillalba** [Mara-Li/mkdocs_embed_file_plugins#4]
When I use Mkdocs-embed-file-plugin with file system (linux) files that are linked throught a linked directory the plugin don't find the file.

**Describe the solution you'd like**
I would like that the plugin works with linked files in linked directories.

**Comment by Mara-Li** [comment]
So you want to list all file contents within a folder?

**Comment by jmvillalba** [comment]
yes. I have a directory that is a linux file system link to the real directory. Then when I try to embed the content of a file in this directory to other file in the same directory, I get a message in the embedded link "file dosen't exists"

I've try to read the code to get a solution, but I can't figure how to.

**Comment by Mara-Li** [comment]
I don't plan to do it myself, because Obsidian doesn't support it and it could bring a lot of slow in the build.

The project is open source so you can code it yourself or anyone can do it!

**Comment by jmvillalba** [comment]
Thanks you for your help.
Well Obsidian support link files and directories out of the vault.

I'll tray to solve it, but I don't see the problem.

Best regards!

**Comment by Mara-Li** [comment]
You can't link to a directory, as far I know. That's why people use folder note.
After you can't embed all file from a directory without using plugin...
